modules/manually-create-identity-access-management.adoc

```diff
@@ -149,108 +141,63 @@ spec:
-spec:
-  secretRef:
-    name: installer-cloud-credentials
-    namespace: openshift-image-registry
-  providerSpec:
-    apiVersion: cloudcredential.openshift.io/v1
     kind: AzureProviderSpec
     roleBindings:
     - role: Contributor
-----
 endif::azure,ash[]
 ifdef::google-cloud-platform[]
-.Sample `CredentialsRequest` object
-[source,yaml]
-----
-apiVersion: cloudcredential.openshift.io/v1
-kind: CredentialsRequest
-metadata:
-  labels:
-    controller-tools.k8s.io: "1.0"
-  name: openshift-image-registry-gcs
-  namespace: openshift-cloud-credential-operator
-spec:
-  secretRef:
-    name: installer-cloud-credentials
-    namespace: openshift-image-registry
-  providerSpec:
-    apiVersion: cloudcredential.openshift.io/v1
     kind: GCPProviderSpec
     predefinedRoles:
     - roles/storage.admin
     - roles/iam.serviceAccountUser
     skipServiceCheck: true
-----
 endif::google-cloud-platform[]
+  ...
+----
 
 . Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object.
 +
-ifdef::ash[] 
-.Sample `CredentialsRequest` object
+.Sample `CredentialsRequest` object with secrets
 [source,yaml]
 ----
 apiVersion: cloudcredential.openshift.io/v1
 kind: CredentialsRequest
 metadata:
-  annotations:
-    include.release.openshift.io/ibm-cloud-managed: "true"
-    include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
-  labels:
-    controller-tools.k8s.io: "1.0"
-  name: openshift-image-registry-azure
-  namespace: openshift-cloud-credential-operator
-spec:
-  providerSpec:
-    apiVersion: cloudcredential.openshift.io/v1
-    kind: AzureProviderSpec
-    roleBindings:
-    - role: Contributor
-  secretRef:
-    name: installer-cloud-credentials
-    namespace: openshift-image-registry
-----
-+
-endif::ash[] 
-
-.Sample `CredentialsRequest` object
-[source,yaml]
-----
-apiVersion: cloudcredential.openshift.io/v1
-kind: CredentialsRequest
-metadata:
-  annotations:
-    include.release.openshift.io/ibm-cloud-managed: "true"
-    include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
-  labels:
-    controller-tools.k8s.io: "1.0"
-  name: openshift-image-registry-azure
+  name: <component-credentials-request>
   namespace: openshift-cloud-credential-operator
+  ...
 spec:
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
+ifdef::aws[]
+    kind: AWSProviderSpec
+    statementEntries:
+    - effect: Allow
+      action:
+      - s3:CreateBucket
+      - s3:DeleteBucket
+      resource: "*"
+endif::aws[]
+ifdef::ash,azure[]
     kind: AzureProviderSpec
     roleBindings:
-    - role: Contributor
+      - role: Storage Account Contributor
+      - role: Storage Blob Data Contributor
```

Comment:
I filtered the 4.13 azure CRs, the role is 'Contributor' only.

jianpingshu@jshu-mac 4.13 % grep "role: " *
0000_26_cloud-controller-manager-operator_14_credentialsrequest-azure.yaml:    - role: Contributor
0000_30_cluster-api_00_credentials-request.yaml:    - role: Contributor
0000_30_machine-api-operator_00_credentials-request.yaml:    - role: Contributor
0000_50_cluster-image-registry-operator_01-registry-credentials-request-azure.yaml:    - role: Contributor
0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml:    - role: Contributor
0000_50_cluster-network-operator_02-cncc-credentials.yaml:    - role: Contributor
0000_50_cluster-storage-operator_03_credentials_request_azure.yaml:    - role: Contributor
0000_50_cluster-storage-operator_03_credentials_request_azure_file.yaml:    - role: Contributor